case_studies/sdss_galaxies/plots.py

```diff
@@ -608,7 +610,7 @@ def compute_data(self, frame: Union[SDSSFrame, SimulatedFrame], encoder, decoder
             resid = (true - recon) / recon.sqrt()
 
             recon_map = tile_map_recon.to_full_params()
-            recon_map.plocs = recon_map.plocs - 0.5
+            recon_map.plocs -= 0.5
```

Comment:
I would suggest doing this adjustment within your plotting code rather than mutating the FullCatalog.